src/components/elements/table/GenericTable.tsx

```diff
@@ -53,7 +53,7 @@ export interface Props<T> {
   headerCellSx?: (def: ColumnDef<T>) => SxProps<Theme>;
   selectable?: 'row' | 'checkbox'; // selectable by clicking row or by clicking checkbox
   isRowSelectable?: (row: T) => boolean;
-  setSelectedRowIds?: (ids: readonly string[]) => void;
+  onChangeSelectedRowIds?: (ids: readonly string[]) => void;
```

Comment:
I renamed this to be more idiomatic "onFoo" to distinguish it from local setters